### DISCUSSION TITLE
Version 0.17 Alpha

### DISCUSSION THREAD

**Discussion by riclolsen** [riclolsen/json-scada#49] [General]
* Default login credentials: username=admin password=jsonscada.
* Fixed IEC60870-5-101/104 commands with select.
* Added Carbone report generator example.
* Alarm beep process can now produce sound when executed as a Windows service.
* Fixed alarm beep for unacked alarm of the same point.
* Disabled Timescaledb telemetry.
* Fixed performance problem in Display Viewer when large point key numbers used (_id).
* MongoDB Community Server updated to 4.4.10.
* MongoDB Compass updated to 1.29.5.
* Chromium updated to 96.0.4664.45.
* Grafana updated to 8.3.0
* Npm package updates.
* DotNet runtime updated to 5.0.12.
* Telegraf runtime updated to 1.20.4.
 
Requirements:

* Windows 10/11 64 bits or Server 2016, Windows PowerShell. At least 10GB of free space in the "C:" drive.
* Administrative rights. Corporative Windows policies may cause problems with the creation of services and the opening of TCP ports.
* Free TCP ports 27017, 5432, 80, 8080, 3000, 9000. Other ports may be required for optional services and protocols.
* If the server already has MongoDB, PostgreSQL, Grafana, Nginx or another webserver, please uninstall, disable or watch out for possible conflicts.
* Do not update previously installed JSON-SCADA. Please uninstall previous JSON-SCADA versions before installing a new version.

<hr /><em>This discussion was created from the release <a href='https://github.com/riclolsen/json-scada/releases/tag/V0.17-alpha'>Version 0.17 Alpha</a>.</em>

**Comment by joyshmitz** [comment]
Hi @riclolsen ,
Happy holiday's
service at address 207.180.242.96 is still broadcasting protocol stream 104 on port 2404?

**Reply by riclolsen** [reply]
Hi!
It probably stopped because of a mongodb update. It should be working now.
I am preparing an image to use with Oracle Cloud free tier of ARM64 VM instance (amazingly always free 4VCPU 24GB RAM 200GB NVME!).
Let me know if you'd like to try it.
Best wishes to you!

**Reply by joyshmitz** [reply]
HI!
Thanks for the answer, I will definitely try

**Reply by joyshmitz** [reply]
Hi @riclolsen,
Have you launched the `json-scada` image for the oracle cloud?
And thanks for the tip about Always Free `Ampere A1`

**Reply by riclolsen** [reply]
I am still working on it.
Can you provide an email or contact me on https://linkedin.com/in/ricardo-olsen/ ?
I do not want to release this image publicly yet.
I will send you the image URL to import on OCI as soon as it is ready.

**Reply by joyshmitz** [reply]
thank you, I sent you a email
